test/test_cams.py

```diff
@@ -77,6 +77,18 @@ def _test_cam(self, name):
 
         self._test_extractor(extractor, model)
 
+    def _test_cam_arbitrary_layer(self, name):
+
+        model = resnet18(pretrained=True).eval()
```

Comment:
Would you mind turning pretrained to False? This does not change the test validity and save both time and storage on CI

Reply:
Sure. I set it since it was also true in the `_test_cam` and `_test_gradcam` methods. Perhaps, those methods could also work with pretrained=False (?).

Reply:
Good point, I'll modify this in a follow-up PR, thanks!